profile/src/main/kotlin/com/imashnake/animite/profile/ProfileScreen.kt

```diff
@@ -200,12 +208,82 @@ fun UserTabs(modifier: Modifier = Modifier) {
                     )
                 )
         ) {
-            Text(
-                text = stringResource(coreR.string.coming_soon),
-                color = MaterialTheme.colorScheme.onSurfaceVariant,
-                style = MaterialTheme.typography.labelLarge,
-                modifier = Modifier.align(Alignment.Center)
-            )
+            // TODO: Use `AnimatedContent`.
```

Comment:
A `HorizontalPager` would be better since it's a tab, and usually you can swipe the tab's content